usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -95,13 +96,13 @@
         "source_database": None,
         "source_table": None,
         "destination_database": "rpt",
-        "swap_table": None,
-        "swap_schema": None,
+        "swap_table": "summary_state_view",
+        "swap_schema": "rpt",
         "partition_column": "duh",
```

Comment:
Was this column always called `duh`. It's just a really bad name and I don't see why we wouldn't have expanded it to something meaningful at a glance. Not worth it now if it would ripple to a bunch of changes in django and api code.

Reply:
Yeh it was always called duh. The help text on the model says `Deterministic Unique Hash`. Not the biggest fan, but also want to minimize differences.